src/cljs/main/broadfcui/page/method_repo/method/exporter.cljs

```diff
@@ -209,4 +210,9 @@
                   (if success?
                     ((:on-export props) workspace-id (ws-common/config->id config))
                     (utils/multi-swap! state (assoc :server-error (get-parsed-response false))
-                                             (dissoc :banner))))}))})
+                                             (dissoc :banner)))
+                  (endpoints/send-metrics-event
+                   "workflow:export:firecloud"
+                   {:success success?
+                    :config (not= (:selected-config @state) :blank)
+                    :userId (:anonymousGroup @user/profile)}))}))}) ;; requiring user deeper causes circular dep
```

Comment:
nit: what is "user deeper"?

Reply:
Ah. What I mean was that requiring `broadfcui.utils.user` in a more generic place, like `broadfcui.utils.ajax`, caused a circular dependency, meaning I have to inject it here, unlike the other non-event-specific information.